resources/ci-builds-mapping.json

```diff
@@ -259,6 +328,88 @@
           }
         }
       },
+      "test_cobertura" : {
```

Comment:
I think naming this entry to `code.coverage` makes more sense to me:

1. cobertura is a Java tool
2. code coverage and test coverage are different metrics in testing literature: https://danashby.co.uk/2019/02/14/code-coverage-vs-test-coverage/

If we want to store the percentage of the code that is covered by tests, then `code coverage` is accurate. But if we want to store the number of tests that exist compared to those we currently know/can imagine, then `test coverage` is the right name.

Reply:
A bit more of context:
- Everything started in https://github.com/elastic/integrations/issues/755

There are different tools in Jenkins for the coverage:
- The `cobertura` coverage plugin -> https://apm-ci.elastic.co/job/apm-server/job/apm-server-mbp/job/master/1087/cobertura/. The pipeline step is [`coverageReport `](https://github.com/elastic/apm-server/blob/master/Jenkinsfile#L298)
- The coverage plugin -> https://apm-ci.elastic.co/job/apm-agent-dotnet/job/apm-agent-dotnet-mbp/job/master/560/coverage/. The pipeline step is [`publishCoverage`](https://github.com/elastic/apm-agent-dotnet/blob/master/Jenkinsfile#L139-L140). __NOTE__: the UI takes a while. The Jenkins plugins is https://plugins.jenkins.io/code-coverage-api/

Since there were different coverage tools in Jenkins, I wanted to use the document key to be specific for the tool. Since `cobertura` was in the API entrypoint I thought it was a good candidate. Then the `coverage` entrypoint could be supported in the future.

OTOH, I used `test_cobertura` similar to the `test_summary`, `test_summary` is the key for the stats related to the JUnit output and it's provided by the `https://apm-ci.elastic.co/blue/rest/organizations/jenkins/pipelines/apm-shared/apm-pipeline-library-mbp/PR-1214/runs/4/blueTestSummary/`. That's the rationale for the `test_***`

What output is stored?
- numerator
- denominator
- ration -> the %

Let me know if those rations make any sense

Reply:
Wdyt about `test_coverage`? It will match the `test_***` pattern.